doc/classes/TextServer.xml

```diff
@@ -326,6 +326,26 @@
 				Returns underline thickness in pixels.
 			</description>
 		</method>
+		<method name="font_get_variation" qualifiers="const">
+			<return type="float">
+			</return>
+			<argument index="0" name="font" type="RID">
+			</argument>
+			<argument index="1" name="tag" type="String">
+			</argument>
+			<description>
+				Returns variation coordinate [code]tag[/code].
+			</description>
+		</method>
+		<method name="font_get_variation_list" qualifiers="const">
+			<return type="Dictionary">
+			</return>
+			<argument index="0" name="fonte" type="RID">
+			</argument>
+			<description>
+				Returns list of supported variation coordinates, each coordinate is returned as [code]Vector3i(min_value,max_value,default_value)[/code].
```

Comment:
Could also have more details as commented in `FontData`. Is the different return format intended?

Reply:
Format is the same `tag: Vector3i(min, max, def)`.